src/gui/AboutDialog.cpp

```diff
@@ -78,20 +78,20 @@ AboutDialog::AboutDialog(QWidget* parent)
 #endif
 
     QString extensions;
-#ifdef WITH_XC_HTTP
-    extensions += "\n- KeePassHTTP";
-#endif
 #ifdef WITH_XC_AUTOTYPE
     extensions += "\n- Auto-Type";
 #endif
-#ifdef WITH_XC_YUBIKEY
-    extensions += "\n- YubiKey";
+#ifdef WITH_XC_BROWSER
+    extensions += "\n- Browser Integration";
+#endif
+#ifdef WITH_XC_HTTP
+    extensions += "\n- Legacy Browser Integration (KeePassHTTP)";
 #endif
```

Comment:
What about `WITH_XC_NETWORKING`?

Reply:
Yeah, agreed. This should be added here.

Reply:
Funnily, these all use #ifdef :laughing:

Reply:
I saw that too after commenting about `if defined` :rofl:

Reply:
#ifdef is much preferred, IMO. #if defined( ) is annoying to read